cmd/terramate/cli/project.go

```diff
@@ -225,24 +226,37 @@ func (p project) checkDefaultRemote() error {
 	)
 }
 
-func (p *project) checkLocalDefaultIsUpdated() error {
+func (p *project) checkRemoteDefaultBranchIsReachable() error {
 	gitcfg := p.gitcfg()
+
+	remoteDesc := fmt.Sprintf("remote(%s/%s)", gitcfg.DefaultRemote, gitcfg.DefaultBranch)
+
+	logger := log.With().
+		Str("HEAD", p.headCommit()).
+		Str(remoteDesc, p.remoteDefaultCommit()).
+		Logger()
+
+	handleError := func(logger zerolog.Logger, debugErrCtx error) error {
+		logger.Debug().Msgf("%s: %v", ErrCurrentHeadIsOutOfDate, debugErrCtx)
```

Comment:
no need to duplicate same message in different levels..

```suggestion
		logger.Debug().Msgf("%v", debugErrCtx)
```